src/main/java/com/hcl/appscan/jenkins/plugin/scanners/Scanner.java

```diff
@@ -38,6 +47,14 @@ public boolean getHasOptions() {
 	public String getTarget() {
 		return m_target;
 	}
+        
+	public boolean getRescan() {
```

Comment:
Can this be named isRescan() since it returns a boolean?

Reply:
Okay, sure.